Tests/FluentMongoDriverTests/FluentMongoDriverTests.swift

```diff
@@ -146,6 +146,10 @@ final class FluentMongoDriverTests: XCTestCase {
             XCTFail("\(error)")
         }
     }
+
+    func testDuplicateIndexNames() throws {
+        XCTAssertNoThrow(try DuplicateIndexMigration().prepare(on: self.db).wait())
```

Comment:
IIUC this won't actually test anything; the original reporter said in Discord that only one index gets created, not that an error is thrown. Is there a way to verify the indexes were actually added? Or alternatively, can we either 1) check for duplicates and throw an error in the schema logic, or 2) ask MongoDB to reject duplicates somehow?